routes/helpers/render.js

```diff
@@ -14,6 +15,11 @@ let render = {
   //  scripts (must not contain sensitive data!)
   template(req, res, view, extraVars, extraJSConfig) {
     let vars = {};
+    
+    // override locale cookie using signupLanguage if present
+    if (req.query.signupLanguage)
+        i18n.setLocale(req, req.query.signupLanguage);
+        vars.signupLanguage = req.query.signupLanguage;
```

Comment:
Nit: You'll either want to surround the entire block in `{` and `}` braces, or unindent the `vars.signupLanguage` statement. The `if` statement ends with the first semicolon, so the continued indentation is semantically incorrect (if meaningless). I would suggest surrounding the entire block with braces, though it functionally makes no difference whether the `vars.signupLanguage` statement always or sometimes executes.